.gitignore

```diff
@@ -1,6 +1,7 @@
-**/*.toolbox/model
-**/*.toolbox/*aux
-**/*.toolbox/*.log
-**/*.toolbox/*.pdf
-**/*.toolbox/*.tex
-**/*.pdf
+.DS_Store
+**/tla/*.toolbox/model
+**/tla/*.toolbox/*aux
+**/tla/*.toolbox/*.log
+**/tla/*.toolbox/*.pdf
+**/tla/*.toolbox/*.tex
+**/tla/*.pdf
```

Comment:
Are these changes required? I thought `**/` meant "in any subdirectory"?

Reply:
no, they're not required, the previous rules would have done fine. I thought that the previous ignore rules were maybe a bit overreaching (if we want to commit other stuff to this repo). With the above change, it's clear that those rules apply to all TLA models.